src/classes/tmControllerTrialFeatureRequest.cls

```diff
@@ -0,0 +1,486 @@
+/*
+*   This class will manage the visualforce page for Trial Org Provisioning. 
+*       @project: IT Commerce
+*       @dev: creinke
+*/
+public with sharing class tmControllerTrialFeatureRequest {
+    public Id tfrId { get; private set; }
+    public Id tfrAccountId { get; private set; }
+    public Boolean showSaved { get; private set; }
+    public Boolean showForm { get; private set; }
+    public String productToRemove { get; set; }
+    public String productToAdd { get; set; }
+    public String templateToAdd { get; set; }
+    
+    /*
+    *   Class Constructor: If the record is new, no ID, then do not show the saved tag. If the page has
+    *       an ID then show it until the user modifies anything on the page. If the tfrDomain is empty
+    *       then throw an error to the page.
+    */
+    public tmControllerTrialFeatureRequest (ApexPages.StandardController controller) {
+        try {
+            displayUpgradeToEditionSection = false;
+            showSaved = false;
+            showForm = true;
+            if (!Test.isRunningTest()) controller.addFields(new String[]{'Account__c'});
+            tmTrialFeatureRequest__c tmTFR = (tmTrialFeatureRequest__c)controller.getRecord();
```

Comment:
Variables should use the `lowerCamelCase` naming convention. Please rename accordingly.